### ISSUE TITLE
Logback configuration error detected with logback 1.1.7

### ISSUE THREAD

**Issue by Kaiyuanliu** [pukkaone/logback-gelf#12]
An unexpected error happened while adding "additionalField" into logback.xml file
![logback-error](https://cloud.githubusercontent.com/assets/8436842/16614511/91add63c-4369-11e6-99fe-e0f773f938be.PNG)

It seems that it is a bug in logback 1.1.7, it doesn't produce those errors when I downgrade to logback 1.1.6. If there is no "additionalField" in the logback.xml file, the version 1.1.7 works as well.

**Comment by pukkaone** [comment]
The error

```
ERROR in ch.qos.logback.core.joran.action.NestedBasicPropertyIA - Unexpected aggregationType AS_BASIC_PROPERTY_COLLECTION
```

was introduced in logback 1.1.7, and appears to be the same as [LOGBACK-1158](http://jira.qos.ch/browse/LOGBACK-1158), which has been fixed for logback 1.1.8 (not released yet).

Until a fixed logback version is released, I can only recommend using logback 1.1.6.

**Comment by mrsarm** [comment]
I had the same issue, and the downgrade to logback 1.1.6 worked.

**Comment by jesuspeon** [comment]
@Kaiyuanliu @mrsarm Logback 1.1.8 is released.

**Comment by pukkaone** [comment]
Upgraded to logback 1.1.8. Released logback-gelf 1.1.10.

**Comment by mrsarm** [comment]
Tested with logback 1.1.8 and logback-gelf 1.1.10: works OK.
